cyhy/mailer/TmailMessage.py

```diff
@@ -135,7 +140,11 @@ def __init__(
 
         """
         # This is the data mustache will use to render the templates
-        mustache_data = {"acronym": entity_acronym, "report_date": report_date}
+        mustache_data = {
```

Comment:
I'm sure you're shocked but~

Reply:
This one is already sorted alphabetically.